pkg/parse_hal/hal_to_service_configs_test.go

```diff
@@ -86,6 +86,10 @@ func TestHalToServiceYAML(t *testing.T) {
 			"orca.yml",
 			services.GetOrca(),
 		},
+		{
```

Comment:
It's awesome that this was the only non-yaml needed to add these tests, I'm getting super on board with these table tests